src/App.js

```diff
@@ -98,6 +153,125 @@ function App() {
         severity: "",
         message: "",
     });
+    const [quizzes] = useState([
+        {
+            id: 'react-basic',
+            title: 'React Basic',
+            category: 'react',
+            level: 'basic',
+            labels: ['frontend']
+        },
+        {
+            id: 'react-middle',
+            title: 'React Middle',
+            category: 'react',
+            level: 'middle',
+            labels: ['frontend']
+        },
+        {
+            id: 'js-basic',
+            title: 'JS Basic',
+            category: 'javascript',
+            level: 'basic',
+            labels: ['frontend', 'backend']
+        },
+        {
+            id: 'js-functions',
+            title: 'JS Functions',
+            category: 'javascript',
+            level: 'basic',
+            labels: ['frontend', 'backend']
+        },
+        {
+            id: 'js-middle',
+            title: 'JS Middle',
+            category: 'javascript',
+            level: 'middle',
+            labels: ['frontend', 'backend']
+        },
+        {
+            id: 'data-structures',
+            title: 'Data structures',
+            category: 'data structure',
+            level: 'middle',
+            labels: ['frontend', 'backend']
+        },
+        {
+            id: 'js-arrays',
+            title: 'JS Arrays',
+            category: 'javascript',
+            level: 'basic',
+            labels: ['frontend', 'backend']
+        },
+        {
+            id: 'js-promises',
+            title: 'JS Promises',
+            category: 'javascript',
+            level: 'middle',
+            labels: ['frontend', 'backend']
+        }
+    ]);
+    const [favoriteQuizzes] = useState([
+        {
+            id: 'react-middle',
+            title: 'React Middle',
+            category: 'react',
+            level: 'middle',
+            labels: ['frontend']
+        },
+        {
+            id: 'js-arrays',
+            title: 'JS Arrays',
+            category: 'javascript',
+            level: 'basic',
+            labels: ['frontend', 'backend']
+        },
+    ]);
+    const [filterVisible, setFilterVisible] = useState(false);
+
+
+    const useFilterState = () => {
```

Comment:
This is a good candidate to put into another file, it could be named `filterState.js` or whatever you see fit.

Reply:
Done ✅